Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
> Does `/usr/include` exist on systems that don't have the CLT installed?

Good question. I don't know. But it doesn't matter, because our `superenv` sets `-isysroot` to the SDK provided by Xcode if the CLT are not installed and the path specified via `-I` is interpreted relative to that.